lib/triggers/ircTrigger.js

```diff
@@ -33,9 +33,7 @@ options: {
   admins: [steamid64,steamid64], //There's no commands yet, but eventually admins will be able to change modes, etc, and users will be able to see who's in chat.
   respectsMute: false,		//Should the relay respect mute? That is, should it stop relaying chat from IRC when you mute it. Default is false.
   defaults: { //default settings everywhere
-    ircOptions: {  //this object will be passed into the irc object without checking it,
-			  						other than sane defaults for the specific items below.
-		  							Full options are available at https://node-irc.readthedocs.org/en/latest/API.html#internal
+    ircOptions: {  //this object will be passed into the irc object without checking it, other than sane defaults for the specific items below. Full options are available at https://node-irc.readthedocs.org/en/latest/API.html#internal
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Line is too long.](https://www.codacy.com/app/node-steam-chat-bot/steam-chat-bot/file/968187204/issues/source?bid=1980036&fileBranchId=2848073#l36)